File: src/main/java/miragefairy2019/mod/modules/fairy/VariantFairy.java
package miragefairy2019.mod.modules.fairy;

import miragefairy2019.mod.api.fairy.FairyType;
import miragefairy2019.mod.lib.multi.ItemVariant;

public class VariantFairy extends ItemVariant
{

	public final FairyType type;

	public VariantFairy(FairyType type)
	{
		this.type = type;
	}

}

File: src/main/java/miragefairy2019/mod/modules/fairy/VariantFairy.kt
package miragefairy2019.mod.modules.fairy

import miragefairy2019.mod.lib.multi.ItemVariant
import miragefairy2019.modkt.impl.fairy.FairyType
import miragefairy2019.modkt.impl.fairy.ColorSet

class VariantFairy(val id: Int, val colorSet: ColorSet, val type: FairyType, val rare: Int, val rank: Int) : ItemVariant()
